bin/pycbc_submit_dax

```diff
@@ -284,33 +273,6 @@ echo "pegasus-run $SUBMIT_DIR/work \$@" > start
 
 chmod 755 start
 
-# Copy planning information into workflow directory so it can be displayed on the results page
-cp pegasus-properties.conf workflow/planning
```

Comment:
Just wanted to comment on this. With removing stuff from pycbc_submit_dax, this section starts to break. I'm also not sure that it is really that important, so I've just taken it out. I could remove only the immediately broken parts, but I'm not sure this is really critical to display in this way.